### PR TITLE
Book model

### PR DESCRIPTION
**Pull request by hellfishg**
## Summary
Basic book model entity

## Trello Card
[Trello card](https://trello.com/b/tV0G7tjv/java-german-asprino)

### CODE REVIEW

File: src/main/java/wolox/training/models/Book.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+    @Column
+    @NotBlank(message = "Pages is mandatory")
+    private String pages;
+
+    @Column
+    @NotBlank(message = "Isbn is mandatory")
+    private String isbn;
+
+    // Getters and setters:
```
Comment (IgnOliveto) [inline src/main/java/wolox/training/models/Book.java:53]: I think that you missed the geter for the id

File: src/main/java/wolox/training/models/Book.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotBlank;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```
Comment (jeymarsonwolox) [inline src/main/java/wolox/training/models/Book.java]: use `GenerationType.SEQUENCE` for postgre, in postgre `GenerationType.AUTO` doesn't generate a sequence for each table

**Review by jeymarsonwolox — CHANGES_REQUESTED** [review]
Great work!!

File: src/main/java/wolox/training/models/Book.java (hunk 2 above)
Comment (IgnOliveto) [reply src/main/java/wolox/training/models/Book.java]: I would use GenerationType.IDENTITY, but the difference is minimal
http://www.ngdeveloper.com/generationtype-identity-vs-generationtype-sequence-vs-generationtype-auto/

Comment (jeymarsonwolox) [reply src/main/java/wolox/training/models/Book.java]: perfect!! 👌

**Review by IgnOliveto — APPROVED** [review]
Nice Job!

**Review by jeymarsonwolox — APPROVED** [review]

**Pull request merged**